examples/hello_triangle_c/CMakeLists.txt

```diff
@@ -34,17 +33,17 @@ elseif(APPLE)
     add_definitions(-DWGPU_TARGET=WGPU_TARGET_MACOS)
     set(OS_LIBRARIES "-framework Cocoa" "-framework CoreVideo" "-framework IOKit" "-framework QuartzCore")
     target_compile_options(${TARGET_NAME} PRIVATE -x objective-c)
-    set(GLFW_LIBRARY glfw)
 else(MSVC)
     add_definitions(-DWGPU_TARGET=WGPU_TARGET_LINUX)
     target_compile_options(${TARGET_NAME} PRIVATE -Wall -Wextra -pedantic)
-    set(GLFW_LIBRARY glfw)
 endif(MSVC)
 
-find_package(glfw3 REQUIRED)
+find_package(glfw3 3.3 REQUIRED)
```

Comment:
You might want to add a hint to this. Now only glfw3.lib is referenced, at least in Visual Studio it cannot find the necessary library file, because the Additional Library Directories is not set. 

I have tried this and that works for me: 
```
find_package(glfw3 3.3 REQUIRED
    HINTS "$ENV{GLFW3_INSTALL_DIR}/lib"
)
```

Reply:
Sure, this sounds reasonable to me, at least for now.

There might be a better way to setup glfw to work with installed binaries and when built from source like we are here, but it's not clear to me right now.

Reply:
Agreed there should be better ways, for now it works at least. We can always review this piece and come to another conclussion.